HISTORY.rst

```diff
@@ -14,6 +14,7 @@ v0.23 (2019-04-04)
 * Support specialized ``ClassVars``, #455 by @tyrylu
 * fix JSON serialization for ``ipaddress`` types, #333 by @pilosus
 * add ``SecretStr`` and ``SecretBytes`` types, #452 by @atheuz
+* Support validators in dataclasses, #454 by @primal100 
```

Comment:
wrong place version is released, Please start a new section.

Reply:
Added to latest section in scolvin:master